operations/bills.md

```diff
@@ -184,12 +245,27 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `ItalianFiscalCode` | string  | optional | Italian fiscal code. |
+| `CustomerID` | string  | optional | ID of the Customer to whom the bill was assigned. |
+| `Address` | [Address](#address)  | optional | Address of the customer. |
+| `LegalIdentifiers` | Dictionary<string, string>  | optional | Legal Identifiers of the customer. |
+| `BillingCode` | string  | optional | ID of the Customer to whom the bill was assigned. |
+| `LastName` | string  | optional | Last name of the customer. |
+| `FirstName` | string  | optional | First name of the customer. |
+| `SecondLastName` | string  | optional | Second last name of the customer. |
+| `TitlePrefix` | string  | optional | Title prefix of the customer. |
+| `ItalianFiscalCode` | string  | deprecated | Italian fiscal code. Deprecated. Use values from LegalIdentifiers |
+| `ItalianLotteryCode` | string  | deprecated | Italian lottery code. Deprecated. Use values from LegalIdentifiers |
 
 #### Bill company data
```

Comment:
Is this the same 'Bill company data' that is referenced from both AssigneeData.Value and OptionalCompanyData? It seems confusing.

Reply:
Yes, the AssigneeData.Value and OptionalCompanyData share the same object definition, though they can have different values